tests/platform_tests/test_reload_config.py

```diff
@@ -71,26 +71,27 @@ def test_reload_configuration_checks(duthosts, rand_one_dut_hostname, localhost,
 
     reboot(duthost, localhost, reboot_type="cold", wait=5)
     logging.info("Reload configuration check")
-    out = duthost.shell("sudo config reload -y", executable="/bin/bash")
+    out = duthost.shell("sudo config reload -y", executable="/bin/bash", module_ignore_errors=True)
```

Comment:
>out [](http://example.com/codeflow?start=4&length=3)

What is the expectation of the return code of each `duthost.shell()`. You need to explicitly check them, either with 0 or known non-zero values.

Reply:
This function is used as a checker to find if config reload is good enough to proceed further. It checks for error message string, is tried until "Retry later" is not thrown. 
Previously there was no return codes and so ansible would continue to give results. However with the recent introduction of error codes without ignoring error param the ansible will crash with a backtrace.
So we are adding ignore errors.
The error code is not considered here because the error message is much more granular and is parsed.